lib/evss/disability_compensation_form/form526_to_lighthouse_transform.rb

```diff
@@ -166,8 +164,50 @@ def transform_service_pay(service_pay_source)
         service_pay_target
       end
 
+      def transform_toxic_exposure(toxic_exposure_source)
+        toxic_exposure_target = Requests::ToxicExposure.new
+
+        gulf_war1990 = toxic_exposure_source['gulfWar1990']
+        gulf_war2001 = toxic_exposure_source['gulfWar2001']
+        if gulf_war1990.present? || gulf_war2001.present?
+          toxic_exposure_target.gulf_war_hazard_service =
+            transform_gulf_war(gulf_war1990, gulf_war2001)
+        end
+
+        toxic_exposure_target
+      end
+
       private
 
+      def transform_gulf_war(gulf_war1990, gulf_war2001)
+        gulf_war1990_value = gulf_war1990&.values&.any?(&:present?) && !none_of_these(gulf_war1990)
+        gulf_war2001_value = gulf_war2001&.values&.any?(&:present?) && !none_of_these(gulf_war2001)
```

Comment:
lovin this ruby chaining elegance :)

we will eventually be adding an i'm not sure option via https://github.com/department-of-veterans-affairs/va.gov-team/issues/73694. is that something we should account for in this story or we would add to another BE ticket?

Reply:
@christinec-fftc  I'm not sure... (pun intended😏) Remind me again of the rules behind "I'm not sure", is it just like treated like "None of these"? i.e. if you select it, you can't/shouldn't select any of the other locations? If so, it would be pretty trivial to just add in right now. If not, then what's the expected result?

Reply:
Summarizing the team discussion in huddle today
- i'm not sure checkboxes on the locations and details pages are for the Veteran's comfort only and should not be sent downstream for processing
- on the location or hazards pages
** if only not sure is selected, ignore it. do not mark yes or no for the question
** if  not sure and location(s) or hazard(s) are selected, same behavior as if only the location(s) or hazard(s) were selected
- on the details pages
** these will be set as `view:` fields, so they should get stripped by the FE submit transformer